components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1754,6 +1754,7 @@ type WorkspaceInstanceStatus struct {
 	Phase        string                       `json:"phase,omitempty"`
 	Repo         *WorkspaceInstanceRepoStatus `json:"repo,omitempty"`
 	Timeout      string                       `json:"timeout,omitempty"`
+	Version      int                          `json:"version,omitempty"`
```

Comment:
Should this be here?

Reply:
Yes. This is the field introduced in #11194 - it just wasn't added to the Go struct definition yet.